<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>362</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>552</width>
    <height>362</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>552</width>
    <height>362</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_Path">
    <property name="whatsThis">
     <string/>
    </property>
    <attribute name="title">
     <string>File Pathes</string>
    </attribute>
    <widget class="QLabel" name="label_Sc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Script Path:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Arch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Archive Path:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ArchPath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ScPath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="open_ArchPath">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_ScPath">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_MF_path">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_MF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Master Files:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_MF_Path">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>120</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_SF_Path">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>150</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Slave Files:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_SF_path">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_Sc_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name Config:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>1</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_param">
    <attribute name="title">
     <string>Ports parameters</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_A">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Port A</string>
     </property>
     <widget class="QLabel" name="label_1A">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2A">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Role</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3A">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cycle install</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_A">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_A">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_B">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Port B</string>
     </property>
     <widget class="QLabel" name="label_1B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Role</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cycle install</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_B">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_B">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_B">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_C">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Port C</string>
     </property>
     <widget class="QLabel" name="label_1C">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2C">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Role</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3C">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cycle install</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_C">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_C">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_C">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_D">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Port D</string>
     </property>
     <widget class="QLabel" name="label_1D">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2D">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Role</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3D">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cycle install</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_D">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_D">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_D">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_auth">
    <attribute name="title">
     <string>Authentication</string>
    </attribute>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="view_password">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
